.gitignore

```diff
@@ -7,7 +7,8 @@ out
 reload/venv
 reload.xml
 .vscode
-
+# Make sure all changes here are local for the developer only.
+config/DevSetup.xml
```

Comment:
I think we should check in an (empty?) one to document this feature and exclude it by adding to .git/info/exclude locally.

I may be missing something but wouldn't know how a DevSetup.xml should look like.